frontend-react/src/pages/submissions/SubmissionTable.tsx

```diff
@@ -37,6 +39,7 @@ const SubmissionTableContent: React.FC<SubmissionTableContentProps> = ({
     const transformDate = (s: string) => {
         return new Date(s).toLocaleString();
     };
+    const featureEvent = `${FeatureName.SUBMISSIONS} | ${EventName.TABLE_FILTER}`;
```

Comment:
Same small, non-blockign point here about hoisting up this variable outside of the component!